CONTRIBUTING.md

```diff
@@ -4,9 +4,10 @@
 
 - Install opam: https://opam.ocaml.org/doc/Install.html
 - Create local switch by running `make init` (this step will take a few minutes, as it will build whole OCaml compiler)
-- Install deps with `opam install . --deps-only --with-test`
-- Build with `make`
-- Run tests with `make test`
+- Build with `make build`
+- Run tests with 
+  - Install test dependencies with `(cd test; yarn)`
```

Comment:
I wonder if we should just run `yarn` from `make`, so there's only a single entry point for commands cc @davesnx

Reply:
From `make init` you mean? I think that makes sense a least!

Reply:
Yes, make should be the only tool you need to interfere to develop